dashboard/db/schema.rb

```diff
@@ -346,13 +346,14 @@
     t.string   "email",              limit: 255, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "school",             limit: 255
     t.string   "code",               limit: 255
+    t.string   "school",             limit: 255
```

Comment:
Yes, it will. It's probably a result of migrating up & down on your machine during development. If you removed and re-added the school field, it would by default go at the end. Probably the easiest fix would be to change it here and then `rake db:schema:load` noting that it will erase your local data. You could also modify your db directly.